theme/views.py

```diff
@@ -499,11 +504,52 @@ def dashboard(request, template="pages/dashboard.html"):
     my_username = request.user.username
     user = User.objects.get(username=my_username)
     my_recent = Variable.recent_resources(user, days=60, n_resources=5)
-
-    context = {'recent': my_recent}
+    today = date.today()
+    beginning = today - timedelta(30)
+
+    my_recommended_resources_list = RecommendedResource.objects\
```

Comment:
Probably should stick this filter on the RecommendedResource object and call it to allow reuse and reduce clutter in the view.

Reply:
I agree.